apps/server/src/modules/tool/external-tool/service/external-tool-validation.service.ts

```diff
@@ -17,6 +20,23 @@ export class ExternalToolValidationService {
 		await this.validateOauth2Config(externalTool);
 
 		this.validateLti11Config(externalTool);
+
+		this.validateLogoSize(externalTool);
+	}
+
+	private validateLogoSize(externalTool: Partial<ExternalTool>): void {
```

Comment:
for readability, please order the methods in first the public methods, then the private ones.

Reply:
But if you have a public method between?
Is this a new style guide point?